obspy/core/util/obspy_types.py

```diff
@@ -411,6 +411,14 @@ def imag(self):
                                       upper_uncertainty=_upper)
 
 
+class ObsPyException(Exception):
+    pass
+
+
+class ZeroSamplingRate(ObsPyException):
+    pass
+
+
```

Comment:
Yes, considered other possibilities and I'm pretty indifferent where it ends up.. feel free to change.. I'm just not feeling much for making it top-level `obspy.exceptions`.